src/dialer.js

```diff
@@ -4,40 +4,69 @@ let inConference = false;
 let currentAtxfer;
 // let sessionIdsInMerge = [];
 
-const setFullName = async () => {
-  const user = await Wazo.getApiClient().confd.getUser(Wazo.Auth.getSession().uuid);
-  const fullName = user.firstName ? `${user.firstName} ${user.lastName}` : user.username;
+/*
+  Permet de récupérer les informations de l'utilisateur connecté 
+  Cette fonction ne sert plus à rien normalement puisqu'on a besoin d'afficher seulement
+  le prénom de l'utilisateur.
+*/
+// const setFullName = async () => {
+//   const user = await Wazo.getApiClient().confd.getUser(
+//     Wazo.Auth.getSession().uuid
+//   );
+
+//   const fullName = user.firstName
+//     ? `${user.firstName} ${user.lastName}`
+//     : user.username;
 
-  $('#user').html(`${fullName} (${user.lines[0].extensions[0].exten})`);
+//   $("#user").html(`${fullName} (${user.lines[0].extensions[0].exten})`);
+// };
+
+/*
+  Permet de récupérer les informations de l'utilisateur connecté
+  puis d'afficher le prénom de l'utilisateur
+*/
+const setGreeter = async () => {
+  const user = await Wazo.getApiClient().confd.getUser(
+    Wazo.Auth.getSession().uuid
+  );
+  const name = user.firstName;
+
+  $(".greeter").html(`Hello ${name} 👋`);
 };
 
-const setMainStatus = status => {
-  $('#dialer .status').html(status);
+/* Permet d'indiquer le statut de l'appel. */
+const setMainStatus = (status) => {
+  $(".status").html(status);
 };
 
-const getNumber = callSession => callSession.realDisplayName || callSession.displayName || callSession.number;
+/* ??? récupère le numéro j'imagine mais euh */
+const getNumber = (callSession) =>
+  callSession.realDisplayName || callSession.displayName || callSession.number;
 
-const getStatus = callSession => {
+/*
+  Permet de récupérer le statut de l'appel
+*/
```

Comment:
english

Reply:
I pushed my code so Jeff can take a look and I forgot to erase my comments. I wrote them for myself to untangle each function and understand the code better. I used to write them in english, I didn't mean these ones to be read 😅